docker/modules/Dockerfile.faux1

```diff
@@ -18,13 +18,50 @@ RUN bin/retry_cmd git clone https://github.com/grafnu/bacnet4j.git --single-bran
 COPY pubber/ pubber/
 RUN pubber/bin/build
 
+# Dockerfile to build older version of SSH and SSL
+FROM daqf/aardvark:latest as ssh_build
+
+RUN $AG update && $AG install wget make build-essential gcc libz-dev ca-certificates
+
+# Build SSH, OpenSSL from source and configure
+RUN wget https://www.openssl.org/source/openssl-1.0.2g.tar.gz && \
```

Comment:
Can you put this and the next RUN command as scripts in the appropriate directory?  This is an awful lot of stuff to have inlined in the Dockerfile.

Reply:
@grafnu I've split them out into separate files as suggested